### ISSUE TITLE
Why you used such a hard dropouts?

### ISSUE THREAD

**Issue by EskimoCold** [Nikossik/MNIST_tutorial#2]
```python
input_layer = Input(shape=input_shape)
fc0 = Dense(64, activation='elu')(input_layer)
dropout0 = Dropout(0.9)(fc0)
fc1 = Dense(128, activation='relu')(dropout0)
dropout1 = Dropout(0.7)(fc1)
fc2 = Dense(32, activation='elu')(dropout1)
output_layer = Dense(10, activation='softmax')(fc2)
```

In this code you firstly dropped out 0.9, then 0.7
Why so hard?
if i wrote this model, i would drop 0.7, then 0.5

**Comment by nikossik** [comment]
## Overfitting 
It is one of the problems of deep Neural Networks (DNN), which consists in the following: the model explains well only the examples from the training sample, adapting to the training examples, instead of learning to classify the examples that did not participate in the training (losing the ability to generalize)
---
## Dropout
![](https://habrastorage.org/web/dd8/171/16f/dd817116fc2348e78272577153e31d2d.jpeg)

- A graphical representation of the Dropout method, taken from the article where it was first introduced. On the left is the neural network before Dropout was applied to it, on the right is the same network after Dropout.
The network shown on the left is used for testing, after learning the parameters.

Networks for training are obtained by dropping out of the network of neurons with probability <img src="https://render.githubusercontent.com/render/math?math=p">, so the probability that the neuron will remain in the network is <img src="https://render.githubusercontent.com/render/math?math=q=1-p">.

Excluded neurons do not contribute to the learning process at any stage of the backpropagation algorithm; therefore, excluding at least one of the neurons is equivalent to training a new neural network.

---
## In a nutshell, Dropout works well in practice because it prevents neurons from interadapting during the learning phase.

**Comment by EskimoCold** [comment]
Ok, thx dude
